### PR TITLE
Adds licence to the Toolkit files

### PR DESCRIPTION
**Pull request by nurv**

### CODE REVIEW

File: bennu-toolkit/src/main/webapp/bennu-toolkit/js/bennu.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
+/*
+ * bennu-angular.js
```
Comment (jcarvalho) [inline bennu-toolkit/src/main/webapp/bennu-toolkit/js/bennu.js]: Wrong filename here

**Comment by jcarvalho** [comment]
Does this take care of the license on the minified file?

**Comment by nurv** [comment]
No, to do that we need to mess around with the pom file. This is just the headers for files.

File: bennu-toolkit/src/main/webapp/bennu-toolkit/js/bennu.js (hunk 1 above)
Comment (nurv) [inline bennu-toolkit/src/main/webapp/bennu-toolkit/js/bennu.js]: A correction to this problem has been submitted.

**Comment by nurv** [comment]
Do you want me to submit a solution with the headers on the minified files?

**Comment by nurv** [comment]
Now it adds a copyright notice on the minification.

**Pull request merged**